src/normal.c

```diff
@@ -131,12 +141,11 @@ static int polynomials[11][5] = {
                   0}  //
 };
 
-//! \brief Access to the tail approximation functions
-//! \param[in] approx The number of the approximation.
-//! \param[in] x The point at which the function is approximated.
+//! \brief Access to the tail approximation polynomial table evaluation
+//! \param[in] approx: The number of the approximation.
+//! \param[in] x: The point at which the function is approximated.
 //! \return ???
```

Comment:
I think you could just say "The value of the function at this point" (and similarly for mid_approx).

Reply:
Went with `The corrected value of the polynomial at x`

Reply:
(And thanks; my grasp of this stuff is shaky enough that I was unclear on how to document things.)